desktop/apps/collect_art/templates/meta.jade

```diff
@@ -1,13 +1,13 @@
-title Buy Art | Artsy
+title Collect Art | Artsy
 - var description = 'An art collection for every budget'
-meta( property='og:title', content='Buy Art | Artsy' )
+meta( property='og:title', content='Collect Art | Artsy' )
 meta( name='description', content= description )
 meta( property='og:description', content= description )
 meta( property='twitter:description', content= description )
-link( rel='canonical', href='#{sd.APP_URL}/buy-art' )
-meta( property='og:url', content='#{sd.APP_URL}/buy-art' )
+link( rel='canonical', href='#{sd.APP_URL}/collect-art' )
+meta( property='og:url', content='#{sd.APP_URL}/collect-art' )
 //- meta( property='og:image', content= asset('/images/og_image.jpg') )
 meta( property='og:type', content='website' )
 meta( property='twitter:card', content='summary' )
 meta( name='viewport', content="width=device-width, initial-scale=1, shrink-to-fit=yes" )
-link( type='text/css', rel='stylesheet', href=asset('/assets/buy_art.css') )
+link( type='text/css', rel='stylesheet', href=asset('/assets/collect_art.css') )
```

Comment:
cc @xtina-starr this can be later (if you even need to), but you might have future issues doing this, vs not `extend`ing from the blank layout, like we discussed yesterday.

If you don't do that, you don't get the `main_layout` JS and CSS. That may be why you needed to require it directly here: https://github.com/artsy/force/pull/1025/files#diff-15a3d1a80fb40d8e8db407b790e65597R1.
You also miss out on some stuff from the `main_layout` JS, which you may not need, but you may find you need, such as [global client stuff](https://github.com/artsy/force/blob/cb5dd3d36db7a5c3ad49b7ce21c3ee13e924aea5/desktop/components/main_layout/client.coffee#L7).